### PR TITLE
🐛 Fixed contributions calendar + updated all packages

### PR DESCRIPTION
**Pull request by BrookJeynes**
#433

### CODE REVIEW

File: src/templates/person.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+import GitHubContributionCalendar from '../components/github-contribution-calendar/github-contribution-calendar';
 import PlayAudio from '../components/play-audio/play-audio';
 import ProfileDescription from '../components/profile-description/profile-description';
 import ProfilePhoto from '../components/profile-photo/profile-photo';
 import PropTypes from 'prop-types';
 import Quote from '../components/quote/quote';
 import SkillsList from '../components/skills-list/skills-list';
 import SocialLinks from '../components/social-links/social-links';
-import { Widget } from 'ssw.rules.widget';
+// import { Widget } from 'ssw.rules.widget';
```
Comment (bradystroud) [inline src/templates/person.js:16]: Comment the widget back in
```suggestion
import { Widget } from 'ssw.rules.widget';
```

File: src/templates/person.js [hunk 2]
```diff
@@ -34,6 +34,7 @@ const Person = ({ pageContext }) => {
   let fullName = '';
   let firstNameOrNickname = '';
   let jobTitle = frontmatter.role;
+  // eslint-disable-next-line no-unused-vars
```
Comment (bradystroud) [inline src/templates/person.js:37]: Remove this
```suggestion
```

File: src/templates/person.js [hunk 3]
```diff
@@ -58,21 +59,21 @@ const Person = ({ pageContext }) => {
...
-  };
+  // const initWidget = () => {
+  //   return (
+  //     <Widget
+  //       token={process.env.WIDGET_GITHUB_PAT}
+  //       author={githubUsername}
+  //       numberOfRules={10}
+  //     />
+  //   );
+  // };
```
Comment (bradystroud) [inline src/templates/person.js:72]: comment this back in

File: src/templates/person.js [hunk 4]
```diff
@@ -58,21 +59,21 @@ const Person = ({ pageContext }) => {
...
 
-  useEffect(() => {
-    githubUsername && setWidgetComponent(initWidget());
-  }, []);
+  // useEffect(() => {
+  //   githubUsername && setWidgetComponent(initWidget());
+  // }, []);
```
Comment (bradystroud) [inline src/templates/person.js:76]: Comment back in

File: src/templates/person.js [hunk 5]
```diff
@@ -146,9 +147,9 @@ const Person = ({ pageContext }) => {
               {socialLinks}
             </div>
             <div className="block md:hidden w-full print-show">{quote}</div>
-            <div className="flex justify-center hidden md:block ">
+            {/* <div className="flex justify-center hidden md:block ">
```
Comment (bradystroud) [inline src/templates/person.js:150]: comment this back in

File: src/templates/person.js [hunk 6]
```diff
@@ -146,9 +147,9 @@ const Person = ({ pageContext }) => {
...
               {githubUsername && WidgetComponent}
-            </div>
+            </div> */}
```
Comment (bradystroud) [inline src/templates/person.js:152]: comment this back in

File: src/templates/person.js [hunk 7]
```diff
@@ -174,11 +175,10 @@ const Person = ({ pageContext }) => {
...
               </>
             )}
-            {/* TODO: Update packages - see https://github.com/SSWConsulting/SSW.People/issues/433 */}
-            {/* {crmData && crmData.gitHubUrl && (
+            {crmData && crmData.gitHubUrl && (
               <GitHubContributionCalendar githubUrl={crmData.gitHubUrl} />
-            )} */}
-            <div className="md:hidden">{githubUsername && WidgetComponent}</div>
+            )}
+            {/* <div className="md:hidden">{githubUsername && WidgetComponent}</div> */}
```
Comment (bradystroud) [inline src/templates/person.js:181]: comment this back in

File: site-config.js [hunk 8]
```diff
@@ -1,5 +1,5 @@
 /*eslint quotes: ["warn", "backtick"]*/
-const path = require(`path`);
+// const path = require(`path`);
```
Comment (bradystroud) [inline site-config.js:2]: Remove this

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Comment by bradystroud** [comment]
@BrookJeynes 
Pipeline is failing at `yarn lint` https://dev.azure.com/ssw/SSW.People/_build/results?buildId=41309&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=80694584-9a00-5514-2ba2-f6dabf815050

To fix
1. run `yarn lint --fix` locally. --fix auto fixes most issues, but sometimes you need to do some manually

**Comment by BrookJeynes** [comment]
> @BrookJeynes Pipeline is failing at `yarn lint` https://dev.azure.com/ssw/SSW.People/_build/results?buildId=41309&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=80694584-9a00-5514-2ba2-f6dabf815050
> 
> To fix
> 
>     1. run `yarn lint --fix` locally. --fix auto fixes most issues, but sometimes you need to do some manually

Done - see latest commits

**Review by bradystroud — APPROVED** [review]

**Pull request merged**